ruff.toml

```diff
@@ -1,15 +1,22 @@
+exclude = [
+  "wagtail/project_template/*",
+  "node_modules",
+  "venv",
+  ".venv",
+  "migrations",
```

Comment:
Would be nice to format migrations too. We previously did when we used black, but it was unintentionally dropped in #11220 because #10551 ignored migrations to ruff's config.

I tried removing `migrations` from here but apparently https://github.com/astral-sh/ruff/issues/9745 causes a lot of unnecessary noise of removing newlines after the migration `class` header, which does not happen with black. Maybe we can revisit when ruff supports it.